src/signals/incident-management/containers/IncidentDetail/components/ChangeValue/index.js

```diff
@@ -237,7 +240,7 @@ const ChangeValue = ({
         <dd data-testid={`meta-list-${type}-value`}>{editForm}</dd>
       ) : (
         <dd data-testid={`meta-list-${type}-value`} className={valueClass}>
-          <DisplayValue data-testid="valuePath">
+          <DisplayValue data-testid="valuePath" highPriority={Boolean(icon)}>
```

Comment:
Boolean props or vars should have a name that depicts their purpose, like `isHighPriority` or `hasHighPrority` or something similar.

Reply:
totally agree!